<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(251, 255, 17)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>121</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QDial" name="attack_dial">
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDial" name="decay_dial">
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDial" name="sustain_dial">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDial" name="release_dial">
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="attack_label">
       <property name="text">
        <string>Attack</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="decay_label">
       <property name="text">
        <string>Decay</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sustain_label">
       <property name="text">
        <string>Sustain</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="release_label">
       <property name="text">
        <string>Release</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLCDNumber" name="sustain_lcd">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="decay_lcd">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="attack_lcd">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLCDNumber" name="release_lcd">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
